.gitignore

```diff
@@ -125,3 +125,5 @@ dmypy.json
 
 # Editor settings
 .vscode
+
+.DS_Store
```

Comment:
Stuff like this, that's not directly related to a given PR, should really go in their own PRs (even if just a tiny one to update the .gitignore file).

Reply:
Thank you.  I have reverted this commit and submitted it as a new pull request (#1863).